src/ripple/protocol/impl/BuildInfo.cpp

```diff
@@ -27,21 +27,17 @@ namespace ripple {
 
 namespace BuildInfo {
 
-char const* getRawVersionString ()
-{
-    static char const* const rawText =
+//------------------------------------------------------------------------------
+char const* const versionString =
```

Comment:
Augment the `gdb` script to execute:

`print 'ripple::BuildInfo::versionString'`